negotiation/src/main/java/it/eng/negotiation/model/Agreement.java

```diff
@@ -30,11 +30,11 @@
 @Document(collection = "agreements")
 public class Agreement {
 
-	@JsonIgnore
-	private String consumerPid;
-	
-	@JsonIgnore
-	private String providerPid;
+//	@JsonIgnore
```

Comment:
If not needed we should remove it from code.
In Offer it is removed, not left commented

Reply:
No problemo